pkg/testcase/secretsencrypt.go

```diff
@@ -102,27 +139,25 @@ func restartServerAndWait(ip, product string) {
 	}
 }
 
-func waitForHashMatch(cpIP, product string) (string, error) {
+func waitForHashMatch(cpIP, product string) (stdOut string, err error) {
```

Comment:
secretEncryptStatus instead of stdOut? stdout is not exactly but kind of a reserved word